protoc-gen-java-optional/src/main/java/org/grpcmock/protoc/plugin/OptionalGenerator.java

```diff
@@ -104,7 +104,7 @@ public static void main(String[] args) {
       String javaPackage
   ) {
     String filePath = javaPackage.replace(".", DIR_SEPARATOR) + DIR_SEPARATOR + fileName + JAVA_EXTENSION;
-    String fullMethodName = protoPackage + "." + messageDescriptor.getName();
+    String fullMethodName = (protoPackage.isEmpty() ? "" : (protoPackage + ".")) + messageDescriptor.getName();
```

Comment:
not sure if this is the best way, but it works. 

I saw a DOT_JOINER [here](https://github.com/salesforce/grpc-java-contrib/blob/f4a9fbd1fad90d04a87e686377d7c4c047a89cf8/jprotoc/jprotoc/src/main/java/com/salesforce/jprotoc/ProtoTypeMap.java#L23), but it is not used with the `protoPackage` (i.e  `protoPackage + "." + e.getName()` is what is used there)